Cython/Utility/ModuleSetupCode.c

```diff
@@ -908,7 +908,11 @@ class __Pyx_FakeReference {
                                                           Py_ssize_t nargs, PyObject *kwnames);
 #else
   #define __Pyx_PyCFunctionFast _PyCFunctionFast
-  #define __Pyx_PyCFunctionFastWithKeywords _PyCFunctionFastWithKeywords
+  #if PY_VERSION_HEX >= 0x030D00A4
```

Comment:
'D' is not easy to distinguish from '0', so we spell the hexver with a lower-case 'd'.
```suggestion
  #if PY_VERSION_HEX >= 0x030d00A4
```